src/components/MapGuide/index.tsx

```diff
@@ -1,101 +1,74 @@
-import { useEffect, useRef, useState } from 'react';
 import Button from '@mui/material/Button';
-import DialogContent from '@mui/material/DialogContent';
-import DialogContentText from '@mui/material/DialogContentText';
-import Paper, { PaperProps } from '@mui/material/Paper';
-import Backdrop, { BackdropProps } from '@mui/material/Backdrop';
-import Draggable from 'react-draggable';
-import CloseIcon from '@mui/icons-material/Close';
+import { styled } from '@mui/material/styles';
+import Tooltip, { TooltipProps, tooltipClasses } from '@mui/material/Tooltip';
 
 import {
-    StyledMapGuideDialog,
     StyledMapGuideButton,
-    StyledDialogTitle,
-    CloseButton,
+    StyledTooltipTitle,
+    StyledTooltipContainer,
 } from './styled';
 
-const PaperComponent = (props: PaperProps) => {
+import React from 'react';
+
+const StyledMapGuideTooltip = styled(
+    ({ className, ...props }: TooltipProps) => (
+        <Tooltip {...props} classes={{ popper: className }} />
+    ),
+)(({ theme }) => ({
+    [`& .${tooltipClasses.tooltip}`]: {
+        backgroundColor: '#0094e2',
+        color: ' #fff',
+        maxWidth: 1050,
+        fontSize: theme.typography.pxToRem(20),
+        border: '1px solid #dadde9',
+        fontFamily: 'Inter, Helvetica, Arial, sans-serif',
+        padding: '0.5rem 2.4rem',
+    },
+    [`& .${tooltipClasses.arrow}`]: {
+        color: '#0094e2',
+    },
+}));
+
+const StyledMapGuideContext: React.FC = () => {
     return (
-        <Draggable
-            handle="#map-guide"
-            cancel={'[class*="MuiDialogContent-root"]'}
-        >
-            <Paper {...props} />
-        </Draggable>
+        <StyledTooltipContainer>
+            <StyledTooltipTitle>
+                Welcome to Global.health Map!
+            </StyledTooltipTitle>
+            <p>
```

Comment:
Instead of regular `p` tag you can use `Typography` component from `mui`. It is then possible to add `sx` or `mb` property to the `Typography` component in order to add margin at the bottom instead of using `<br />` tags. You can see more here: https://mui.com/material-ui/react-typography/#main-content